mobile/src/navigation/chat/chat.navigation.tsx

```diff
@@ -0,0 +1,37 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import React, { FC } from 'react';
+
+import { ChatScreenName } from '~/common/enums/enums';
+import { ChatNavigationParamList } from '~/common/types/types';
+import { useAppNavigate, useCallback, useFocusEffect } from '~/hooks/hooks';
+
+import { NAVIGATION_ITEMS, SCREEN_OPTIONS } from './common/constants/constants';
+
+const Chat: FC = () => {
+  const NativeStack = createNativeStackNavigator<ChatNavigationParamList>();
+  const navigation = useAppNavigate();
+
+  useFocusEffect(
+    useCallback(() => {
+      navigation.setOptions({
+        headerShown: true,
+      });
+    }, []),
+  );
```

Comment:
Just useEffect will be enough, we don't want to set these options again when we enter screen